core-java-modules/core-java-lang-6/src/main/java/com/baeldung/finalstatic/Bike.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.finalstatic;
```

Comment:
Let's rename the package to `staticfinal`

Reply:
Thanks @Maiklins, resolved.